libs/crypto/tests/unit/mcl_dkg_tests.cpp

```diff
@@ -0,0 +1,271 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "crypto/mcl_dkg.hpp"
+#define MCLBN_FP_UNIT_SIZE 4
+
+#include "gtest/gtest.h"
+
+#include <cstdint>
+#include <iostream>
+#include <ostream>
+
+using namespace fetch::crypto::mcl;
+using namespace fetch::byte_array;
+
+TEST(MclTests, BaseMcl)
+{
+  const char *aa = "12723517038133731887338407189719511622662176727675373276651903807414909099441";
```

Comment:
What are these? Variable names should be descriptive if possible

Reply:
They correspond to points on the elliptic curve.